docs/maya/ui_overview.md

```diff
@@ -50,42 +54,70 @@ The AdonisFX Menu provides access to the options in the shelf and other more adv
 
 ### Create Section
 
-#### Locators and Sensors
-
-There is an item to create each locator and sensor type (Figures 3 and 4). Also, there are option boxes to launch a simple UI to provide a custom name for the node to create (Figure 5).
+#### Locators
 
-<figure style="width: 49%;float: left" markdown>
+<figure style="width: 50%;" markdown>
   ![AdonisFX Menu Create Locators](images/adonis_menu_locators.png)
   <figcaption><b>Figure 3</b>: AdonisFX Menu Create Locators section.</figcaption>
 </figure>
 
-<figure style="width: 49%;float: right" markdown>
+- **Position**. Creates an AdnLocatorPosition node.
+- **Distance**. Creates an AdnLocatorDistance node.
+- **Rotation**. Creates an AdnLocatorRotation node.
```

Comment:
Can we reuse the descriptions used in the shelf icons? Also, I would add a cross reference to the icon saying something like "This item corresponds to the shelf button XXX.png"

Reply:
Done! Applied the additional comment to all menu items that appear on the shelf.